matchms/filtering/SpectrumProcessor.py

```diff
@@ -305,12 +298,12 @@ def __str__(self):
         pd.set_option('display.max_columns', 4)
         pd.set_option('display.width', 1000)
         report_str = f"""\
------ Spectrum Processing Report -----
-Number of spectrums processed: {self.counter_number_processed}
-Number of spectrums removed: {sum(self.counter_removed_spectrums.values())}
-Changes during processing:
-{str(self.to_dataframe())}
-"""
+            ----- Spectrum Processing Report -----
+            Number of spectrums processed: {self.counter_number_processed}
+            Number of spectrums removed: {sum(self.counter_removed_spectrums.values())}
+            Changes during processing:
+            {str(self.to_dataframe())}
+            """
```

Comment:
This didn't work for me and will create a string with many tabs in front of each line.

Reply:
Ah good catch! Now I understand why it was weirdly formatted before...

Reply:
And the second solution is nice, did not know that that was possible in python